### PR TITLE
refactor(CLI): Use the new kubeconfig functions in CP CLI commands

### PR DESCRIPTION
**Pull request by ewollesen**
Refactors the curated packages CLI commands to use the new functions introduced in ac5db7f. The non curated packages commands were not refactored, because I'm not 100% sure it's safe to do so. I'll leave that exercise for a later PR.

This helps simplify our k8s go client creation for CP CLI commands, which will be coming in a follow-up commit.

### CODE REVIEW

**Comment by ewollesen** [comment]
/hold 

Waiting for PR #3462

**Comment by chrisdoherty4** [comment]
The second commit on this looks good. Ping me for the lgtm when ready.

**Review by chrisdoherty4 — APPROVED** [review]
lgtm

**Comment by ewollesen** [comment]
> lgtm

TIL you don't need the `/` in front of commands. How had I not noticed that before?

**Comment by chrisdoherty4** [comment]
> TIL you don't need the / in front of commands. How had I not noticed that before?

You do need the `/`. As a reviewer, you can either comment with `/lgtm` or click 'Approve' and prow will apply the `lgtm` label. 

I 'lgtm' in my approve comment out of habbit.

**Pull request merged**
